R-package/R/lstm.R

```diff
@@ -0,0 +1,589 @@
+require(mxnet)
+
+# lstm cell symbol
+lstm <- function(num.hidden, indata, prev.state, param, seqidx, layeridx, dropout=0) {
```

Comment:
Oh I see you have documentation for other exposed APIs. Ignore this comment. Thanks.